components/NavigationDrawer/src/private/MDCBottomDrawerContainerViewController.m

```diff
@@ -0,0 +1,657 @@
+/*
+ Copyright 2018-present the Material Components for iOS authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#import "MDCBottomDrawerContainerViewController.h"
+
+#import "../MDCBottomDrawerHeader.h"
```

Comment:
Do we need the `../`?

Reply:
AFAIK, yes. but i can easily be proved wrong if someone knows more

Reply:
I believe you should have scope to your component/src directory.
See private/MDCAlertControllerView+Private.m which directly import from its own directory as well as the components main src directory without relative paths.

Reply:
done! apparently our .bzl file already includes the private and the src all in one.